### PR TITLE
Update kms sample to use libraries-bom.

### PR DESCRIPTION
**Pull request by kurtisvg**
Hoping to resolve the diamond dep conflict that #1957 is experiencing.

### CODE REVIEW

**Comment by kurtisvg** [comment]
@gguuss friendly ping for review

**Comment by lesv** [comment]
I'm going to approve as admin unless @gguuss comes in and stops me before Kokoro is done.

**Comment by lesv** [comment]
J11 failure

**Comment by lesv** [comment]
```
[ERROR] Tests run: 11, Failures: 0, Errors: 8, Skipped: 0, Time elapsed: 2.791 s <<< FAILURE! - in com.example.SnippetsIT
[ERROR] restoreCryptoKeyVersion_restores(com.example.SnippetsIT)  Time elapsed: 0.284 s  <<< ERROR!
com.google.api.gax.rpc.ResourceExhaustedException: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.
Caused by: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.

[ERROR] setPrimaryVersion_createKeyAndSetPrimaryVersion(com.example.SnippetsIT)  Time elapsed: 0.048 s  <<< ERROR!
com.google.api.gax.rpc.ResourceExhaustedException: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.
Caused by: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.

[ERROR] addAndRemoveMemberToCryptoKeyPolicy_addsDisplaysAndRemoves(com.example.SnippetsIT)  Time elapsed: 0.576 s  <<< ERROR!
com.google.api.gax.rpc.ResourceExhaustedException: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.
Caused by: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.

[ERROR] destroyCryptoKeyVersion_destroys(com.example.SnippetsIT)  Time elapsed: 0.058 s  <<< ERROR!
com.google.api.gax.rpc.ResourceExhaustedException: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.
Caused by: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.

[ERROR] addAndRemoveMemberToKeyRingPolicy_addsDisplaysAndRemoves(com.example.SnippetsIT)  Time elapsed: 0.67 s  <<< ERROR!
com.google.api.gax.rpc.ResourceExhaustedException: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.
Caused by: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.

[ERROR] enableCryptoKeyVersion_enables(com.example.SnippetsIT)  Time elapsed: 0.043 s  <<< ERROR!
com.google.api.gax.rpc.ResourceExhaustedException: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.
Caused by: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.

[ERROR] disableCryptoKeyVersion_disables(com.example.SnippetsIT)  Time elapsed: 0.044 s  <<< ERROR!
com.google.api.gax.rpc.ResourceExhaustedException: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.
Caused by: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.

[ERROR] com.example.SnippetsIT  Time elapsed: 0.221 s  <<< ERROR!
com.google.api.gax.rpc.ResourceExhaustedException: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.
Caused by: io.grpc.StatusRuntimeException: RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer 'project_number:779844219229'.

[ERROR] Errors:
[ERROR]   SnippetsIT.addAndRemoveMemberToCryptoKeyPolicy_addsDisplaysAndRemoves » ResourceExhausted
[ERROR]   SnippetsIT.addAndRemoveMemberToKeyRingPolicy_addsDisplaysAndRemoves » ResourceExhausted
[ERROR]   SnippetsIT.com.example.SnippetsIT » ResourceExhausted io.grpc.StatusRuntimeExc...
[ERROR]   SnippetsIT.destroyCryptoKeyVersion_destroys » ResourceExhausted io.grpc.Status...
[ERROR]   SnippetsIT.disableCryptoKeyVersion_disables » ResourceExhausted io.grpc.Status...
[ERROR]   SnippetsIT.enableCryptoKeyVersion_enables » ResourceExhausted io.grpc.StatusRu...
[ERROR]   SnippetsIT.restoreCryptoKeyVersion_restores » ResourceExhausted io.grpc.Status...
[ERROR]   SnippetsIT.setPrimaryVersion_createKeyAndSetPrimaryVersion » ResourceExhausted
[ERROR] Tests run: 17, Failures: 0, Errors: 8, Skipped: 0
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-failsafe-plugin:3.0.0-M3:verify (default) on project kms-samples: There are test failures.
[ERROR]
[ERROR] Please refer to /tmpfs/src/github/java-docs-samples/kms/target/failsafe-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

File: kms/pom.xml [hunk 1]
```diff
@@ -21,11 +21,27 @@
...
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
   <dependencies>
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-kms</artifactId>
-      <version>1.35.1</version>
```
Comment (gguuss) [inline kms/pom.xml]: 1.36.1?

**Comment by kurtisvg** [comment]
Admin merging - flaky tests are tracked in #1764.

**Pull request merged**